packages/contentstack/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@contentstack/cli",
   "description": "Command-line tool (CLI) to interact with Contentstack",
-  "version": "1.13.0",
+  "version": "1.13.1",
```

Comment:
Hey @shafeeqd959 , version already bumped to 1.13.1 for GCP release. https://github.com/contentstack/cli/pull/1277